.github/policies/resourceManagement.yml

```diff
@@ -0,0 +1,156 @@
+id:
+name: GitOps.PullRequestIssueManagement
+description: GitOps.PullRequestIssueManagement primitive
+owner:
+resource: repository
+disabled: false
+where:
+configuration:
+  resourceManagementConfiguration:
+    scheduledSearches:
+    - description:
+      frequencies:
+      - hourly:
+          hour: 1
+      filters:
+      - isIssue
+      - isOpen
+      - hasLabel:
+          label: 'Needs: Author Feedback'
+      - hasLabel:
+          label: 'Status: No Recent Activity'
+      - noActivitySince:
+          days: 3
+      - isNotLabeledWith:
+          label: Service Bug
```

Comment:
What's the difference between this label and `bug`?

Reply:
We use Service Bug to signal that we are not the team responsible for the bug since it is a very common scenario.